### PR TITLE
Merge beta

### PR DESCRIPTION
**Pull request by steverydz**
## Done

- Branch from master
- Merge beta-1.7.0 into branch
- Resolve conflicts

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- All code has been QA'd so it's mostly a case of making sure nothing looks broken

### CODE REVIEW

File: templates/cloud/index.html [hunk 1]
```diff
@@ -0,0 +1,510 @@
+{% extends "cloud/base_cloud.html" %}
```
Comment (nottrobin) [inline templates/cloud/index.html]: this page doesn't exist any more right? redirects to `/openstack`. So I'm assuming this file shouldn't exist?

File: templates/containers/lxd.html [hunk 2]
```diff
@@ -77,9 +77,6 @@ <h2 class="p-muted-heading u-align--center">Supported guest operating systems</h
       <li class="p-inline-images__item">
```
Comment (nottrobin) [inline templates/containers/lxd.html]: Does this page exist in the new IA? Shouldn't this maybe be deleted too? If not, just ignore me.

File: templates/download/cloud/build-openstack.html [hunk 3]
```diff
@@ -0,0 +1,226 @@
+{% extends "download/_base_download.html" %}
```
Comment (nottrobin) [inline templates/download/cloud/build-openstack.html]: Another page being created here - is this the same thing? Should this remain deleted 'cos it's not in the new IA?

**Comment by steverydz** [comment]
@nottrobin Thanks - it's expected to fail at the moment as there were some links to pages we haven't made yet. We'll get that cleaned up as part of the QA.

**Pull request merged**